cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -235,6 +235,40 @@ def _add_storage_widgets(self, metrics, storages_list, namespace, dimension_name
             widgets_list.append(graph_widget)
         return widgets_list
 
+    def _add_fsx_widgets(self, storages_list):
```

Comment:
Minor: when adding a widget to a dashboard, it could be better to put a screenshot of the widget to the PR description

Reply:
Thanks. I will improve.

BTW, this change does not add new widget. It adds more lines to existing graphs